tests/.test_configs/db_config.json

```diff
@@ -1,7 +1,7 @@
 {
     "db_connection_string": "mysql+mysqldb://rdr:rdr!pwd@127.0.0.1/?charset=utf8",
     "backup_db_connection_string": "mysql+mysqldb://rdr:rdr!pwd@127.0.0.1/?charset=utf8",
-    "unittest_db_connection_string": "mysql+mysqldb://root@127.0.0.1:9306/?charset=utf8",
+    "unittest_db_connection_string": "<overridden in tests>",
```

Comment:
Does this get evaluated somehow or is it just there for a comment?

Reply:
It's mostly there as a placeholder. To get the code that overrides the connection string to work there needs to be something in the config file for it to override.